# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1319024041 -3600
#      Wed Oct 19 12:34:01 2011 +0100
# Node ID c943be5f929da853b3705283d0aa08bdde227734
# Parent  30c4bd2592dd8bb8a7c1037dd773c0a77a8d3813
Make unused ARPACK code F77 compliant.

* arpack/util/second.f (ETIME): Change declaration to EXTERNAL.

diff --git a/libcruft/arpack/util/second.f b/libcruft/arpack/util/second.f
--- a/libcruft/arpack/util/second.f
+++ b/libcruft/arpack/util/second.f
@@ -16,17 +16,17 @@
 *     .. Local Scalars ..
       REAL               T1
 *     ..
 *     .. Local Arrays ..
       REAL               TARRAY( 2 )
 *     ..
 *     .. External Functions ..
       REAL               ETIME
-      INTRINSIC          ETIME
+      EXTERNAL           ETIME
 *     ..
 *     .. Executable Statements ..
 *
 
       T1 = ETIME( TARRAY )
       T  = TARRAY( 1 )
 
       RETURN
